pkgs/by-name/ko/komga/package.nix

```diff
@@ -32,8 +32,11 @@ stdenvNoCC.mkDerivation rec {
     description = "Free and open source comics/mangas server";
     homepage = "https://komga.org/";
     license = lib.licenses.mit;
-    platforms = jdk17_headless.meta.platforms;
-    maintainers = with lib.maintainers; [ govanify ];
+    platforms = jdk23_headless.meta.platforms;
+    maintainers = with lib.maintainers; [
+      govanify
+      tebriel
+    ];
```

Comment:
IMO, this would be a third commit: `komga: add tebriel to maintainers` but it's probably fine as-is.